### DISCUSSION TITLE
用U盘工具写最新镜像报错

### DISCUSSION THREAD

**Discussion by swordpp** [hzyitc/armbian-onecloud#192] [Q&A]
你好，我下载了最新的24.5镜像，然后使用U盘工具写U盘的时候总是出错，请问这是为什么？更换了好几个工具，都是一样，没有一个能写成功。
我也试了更换电脑，还是一样，写到即将结束的时候就报错。
![企业微信截图_17109835735217](https://github.com/hzyitc/armbian-onecloud/assets/131244293/21897b1b-2eb7-43dc-b142-22f22e227606)
我试了几乎所有镜像文件。
而且使用这个镜像去刷玩客云也报错，刷不进去。
不知道是什么问题。

**Comment by hzyitc** [comment]
换个u盘试试吧。
